airbyte-integrations/bases/connector-acceptance-test/connector_acceptance_test/tests/test_full_refresh.py

```diff
@@ -36,24 +37,26 @@ def primary_keys_only(record, pks):
 
 @pytest.mark.default_timeout(20 * 60)
 class TestFullRefresh(BaseTest):
-    def test_sequential_reads(
-        self,
-        inputs: ConnectionTestConfig,
-        connector_config: SecretDict,
-        configured_catalog: ConfiguredAirbyteCatalog,
-        docker_runner: ConnectorRunner,
-        detailed_logger: Logger,
+    def assert_emitted_at_increase_on_subsequent_runs(self, first_read_records, second_read_records):
+        first_read_records_data = [record.data for record in first_read_records]
+        assert first_read_records_data, "At least one record should be read using provided catalog"
+
+        first_read_records_emitted_at = [record.emitted_at for record in first_read_records]
+        max_emitted_at = max(first_read_records_emitted_at)
+
+        second_read_reords_emitted_at = [record.emitted_at for record in second_read_records]
+
+        min_emitted_at = min(second_read_reords_emitted_at)
+
+        assert max_emitted_at < min_emitted_at, "emitted_at should increase on subsequent runs"
```

Comment:
```suggestion
        assert max_emitted_at_first_read < min_emitted_at_second_read, "emitted_at should increase on subsequent runs"
```
I find it a bit confusing to read `max < min`

Reply:
great suggestion